### ISSUE TITLE
Grad overflow and null validation loss

### ISSUE THREAD

**Issue by NewDriverLee** [IntelLabs/academic-budget-bert#33]
In the first epoch of pretraining, grad overflow happened in every iteration. Also, the evaluation loss of some epochs is null, after about the 17th epoch. But some epochs still exhibit valid evaluation loss. Anybody met the same issue? Will it lead to an incorrect pretraining result?
